tldr/cross_file_calls.py

```
@@ -3939,3 +4061,252 @@ def _build_php_call_graph(
                                     if name == method:
                                         graph.add_edge(rel_path, caller_func, file_path, method)
                                         break
+
+
+def _extract_elixir_file_calls(file_path: Path, root: Path) -> dict[str, list[tuple[str, str]]]:
```

Ruff (0.15.5)

[warning] Too many statements (75 > 50)

(PLR0915)